docs/advanced-features.md

```diff
@@ -0,0 +1,65 @@
+## Context
```

Comment:
This was in an unreachable doc. Moved here with the new caching documentation, and deleted the unreachable file.